index.php

```diff
@@ -23,38 +39,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Skate Plazas Worldwide</title>
 </head>
-<body>
-<header>
+<div>
+<header id="header">
         <h1>Skate Plazas Worldwide</h1>
 </header>
 <main>
     <div class=containers>
     <?php
-    foreach($skatePlazas as $key => $plaza) {
-        $key++;
-        echo "<div class=plazas>";
-            echo "<h1>{$plaza['name']}</h1>";
-                echo "<div class=box style='background-image: url({$plaza['photo']})'>";
-                echo "</div>";
-            echo "<p>Location: {$plaza['city']}, {$plaza['country']}</p>";
-            echo "<p>Date of construction: {$plaza['dob']}</p>";
+
+    /**
+     * Input database array, foreach it and display each item in a nice little box
+     *
+     * @param array $skatePlazas
+     * @return string
+     */
+    function mainPageDisplay (array $skatePlazas): string
```

Comment:
Perfect! Same comment as above, this should now sit in a separate PHP file with the other two functions.

You'll also need to write unit tests for this `mainPageDisplay` function (but not the other two since they interact with the database)!